tests/test_regr.py

```diff
@@ -122,26 +122,25 @@ def test_check_package___init__(finalize_linter: PyLinter) -> None:
     assert checked == ["__init__"]
 
 
-# TODO: 3.0: Test are broken because of property shenanigans of config attribute
-# Re-enable after some of the old attributes have been removed after deprecation period
-@pytest.mark.xfail
 def test_pylint_config_attr() -> None:
     mod = astroid.MANAGER.ast_from_module_name("pylint.lint.pylinter")
     pylinter = mod["PyLinter"]
     expect = [
         "_ArgumentsManager",
         "object",
+        "_MessageStateHandler",
         "ReportsHandlerMixIn",
-        "BaseTokenChecker",
         "BaseChecker",
         "_ArgumentsProvider",
     ]
     assert [c.name for c in pylinter.ancestors()] == expect
     assert list(astroid.Instance(pylinter).getattr("config"))
     inferred = list(astroid.Instance(pylinter).igetattr("config"))
     assert len(inferred) >= 1
-    assert inferred[0].root().name == "argparse"
-    assert inferred[0].name == "Namespace"
+    # TODO:Test are broken because of property shenanigans of config attribute
+    # Re-enable after some of the old attributes have been cleanup up
+    # assert inferred[0].root().name == "argparse"
```

Comment:
This is ``uninferrable`` right now.

Reply:
Thathttps://github.com/PyCQA/pylint/blob/694c533fb34f18bb6202e3f70444e9464f544f3e/pylint/config/arguments_manager.py#L84-L87

I guess we have trouble inferring this? I would be okay with removing this tests. It doesn't seem useful anyway:
https://github.com/PyCQA/pylint/commit/fc0ed972b474e63c0d500d7f2d865beebdb12d51

It's very old and useless now